src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/Reaction/AngularDistribution/Equiprobable/Builder/src/values.hpp

```diff
@@ -0,0 +1,12 @@
+template< typename Range,
+          utility::Require< true, utility::is_range, Range > = true >
+Builder& values( Range&& bins ){
+  try{ 
+    this->cosineBins_ = dvCB< std::array< double, 33 > >( 
+            details::make_array< 33 > (bins ) );
+    return *this;
+  } catch( details::verify::exceptions::InvalidCosine& c ){
+    Log::info( "Trouble encountered constructing cosine bins" );
+    throw;
+  }
```

Comment:
Does make_array throw exceptions? If so, maybe catch them here?

Reply:
No, it doesn't—at least we don't actively catch/throw any exceptions.